cookbook/form/data_transformers.rst

```diff
@@ -111,29 +111,100 @@ for converting to and from the issue number and the ``Issue`` object::
 Using the Transformer
 ---------------------
 
-Now that you have the transformer built, you just need to add it to your
-issue field in some form.
+As seen above our transformer requires an instance of an object manager. While for most
+use-cases using the default manager is fine we will let you pick the manager by it's name.
```

Comment:
We avoid the first person perspective in the docs. So we should reword the sentence a bit like the following:

> While for most use-cases it is sufficient to use the default entity manager, you will sometimes need to explicitly choose the one to use. To achieve this, you can use a factory::